### PR TITLE
Add isEmpty validation rule

### PR DESCRIPTION
**Pull request by morpheus7CS**
Addresses the discussion in issue #81.

Kind regards,
g

### CODE REVIEW

**Comment by oussamahamdaoui** [comment]
Thank you, there is a lint error that is blocking the merge:
"line 14, col 39, Error - Unexpected whitespace before property length. (no-whitespace-before-property)" in /testFunctions/types/array.js can you fix it an push again?

File: src/testFunctions/types/array.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ const array = {
     });
     return ret;
   },
-  notEmpty: val => val.length !== 0,
+  notEmpty: (val, itShouldNotBeEmpty) => (val.length !== 0) === itShouldNotBeEmpty,
+  isEmpty: (val, itShouldBeEmpty) => (val. length === 0) === itShouldBeEmpty,
```
Comment (oussamahamdaoui) [inline src/testFunctions/types/array.js]: the space is making the linter fail

**Comment by morpheus7CS** [comment]
Should be fine now, not sure how that one sneaked in there.

Kind regards,
g

**Comment by oussamahamdaoui** [comment]
Can you please add a test for the `isEmpty` in the array example to keep the code covrage 100%

**Comment by morpheus7CS** [comment]
Hey @oussamahamdaoui,

I've tried looking at this a couple of different ways, but (completely honestly) still have no clue as to what should be added or where to increase the code coverage. Would you mind stepping in?

Thanks,
g

**Comment by oussamahamdaoui** [comment]
Hey, thanks ill look at it tonight.

**Pull request closed without merging**